### DISCUSSION TITLE
Errors in devtools:check()  (Version: 2022.07.20)

### DISCUSSION THREAD

**Discussion by SudoWodo** [SudoWodo/StochOptim#31] [Q&A]
Prof. Werner reported errors in one of the example which popped up in `devtools:check()` with the latest version (Version: 2022.07.20) of the package. Do let me know if others are getting errors as well.

**Comment by nashjc** [comment]
I get just 1 NOTE, but it does seem to be problematic!

❯ checking package dependencies ... NOTE
   Packages suggested but not available for checking:
     'pso', 'DEoptim', 'DEoptimR', 'GenSA', 'adagio'

**Reply by SudoWodo** [reply]
I am neither getting any error, warnings or notes. For testing purposes I removed some of the installed packages to see if `sopm()` would skip them and continue with other packages so as to test the code and than reinstalled them later. So may be that could be one of the reasons why I am not getting notes or errors.

I'll look into : 
```
checking package dependencies ... NOTE
Packages suggested but not available for checking:
```
Thank you for informing.

**Reply by SudoWodo** [reply]
oh I see what you mean, so basically when someone is installing `StochOptim` afresh and doesn't have all the packages installed which are mentioned in the suggested list of DESCRIPTION file they will get the warning for the packages that aren't installed:- 
```
checking package dependencies ... NOTE
Packages suggested but not available for checking:
```
We added these packages to the suggested section so that we don't get issue mentioned in #19 (test() vs check() ) .
